Components/Cards/card.html

```diff
@@ -0,0 +1,374 @@
+            >
+              <div
+                class="text__card___container d-flex col p-sm m-1 rounded box-shadow"
+              >
+                <h1 class="txt-3xl txt-bold">Card Title</h1>
+                <h6 class="txt-light">Card Subtitle</h6>
+                <p class="txt-regular">
+                  Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+                  Perferendis recusandae magni corporis rerum earum amet ducimus
+                  at eligendi reprehenderit voluptates.
+                </p>
+              </div>
+            </div>
+
+            <div class="component__carbon--container mt-2">
+              <code>
+                <iframe
+                  src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=one-light&wt=none&l=htmlmixed&width=974.5&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522text__card___container%2520d-flex%2520col%2520p-sm%2520m-1%2520rounded%2522%253E%250A%2520%2520%253Ch1%2520class%253D%2522txt-3xl%2520txt-bold%2522%253ECard%2520Title%253C%252Fh1%253E%250A%2520%2520%253Ch6%2520class%253D%2522txt-light%2522%253ECard%2520Subtitle%253C%252Fh6%253E%250A%2520%2520%253Cp%2520class%253D%2522txt-regular%2522%253E%250A%2520%2520%2520%2520Lorem%2520ipsum%2520dolor%2520sit%252C%2520amet%2520consectetur%2520adipisicing%2520elit.%250A%2520%2520%2520%2520Perferendis%2520recusandae%2520magni%2520corporis%2520rerum%2520earum%2520amet%2520ducimus%250A%2520%2520%2520%2520at%2520eligendi%2520reprehenderit%2520voluptates.%250A%2520%2520%253C%252Fp%253E%250A%253C%252Fdiv%253E"
+                  style="border: 0; transform: scale(1); overflow: hidden"
+                  sandbox="allow-scripts allow-same-origin"
+                >
+                </iframe>
+              </code>
+            </div>
+          </div>
+
+          <div class="component__examples">
+            <h4 id="verticalCard" class="mb-1">Example of Vertical Cards</h4>
+
+            <p class="txt-light mb-1">
+              You can use the following snippet and customize your own
+              <span class="txt-primary">Vertical Product card</span> component.
+            </p>
+
+            <div
+              class="component__examples--container p-1 rounded d-flex align-center wrap gap-2"
+            >
+              <div
+                class="card__vertical__container d-flex col box-shadow rounded m-sm"
+              >
+                <span class="txt-badge rounded card__badge">New Offer</span>
+                <div class="card__icon">
+                  <i class="fas fa-heart fa" aria-hidden="true"></i>
+                </div>
+
+                <img
+                  src="/Assets/camera.jpg"
+                  alt="camera"
+                  class="img-responsive card-img"
+                />
+
+                <div class="text-container flex-1 d-flex col py-sm">
```

Comment:
have some padding in between text and border of the card.

Reply:
I have added it , `py-sm` is a utility class which gives `padding : .5rem 0rem;

Reply:
I was talking about padding left and right because the text was touching the border of the card and padding will make it easy to read.